src/main/java/backend/Season.java

```diff
@@ -29,6 +30,15 @@ public int getCurrentRound() {
     }
 
     /**
+     * Sets the current round.
+     *
+     * @param currentRound The round to be set to current Round
+     */
+    public void setCurrentRound(int currentRound) {
+        this.currentRound = currentRound;
```

Comment:
For later: add a check to see if the current round is between 1 and the amount of races in the season